Minecraft.Client/Blocks/Lava.cs

```diff
@@ -4,4 +4,14 @@ public class Lava : Block
     {
         public Lava() : base(BlockType.Lava) { }
     }
+
```

Comment:
I'm going to play a little bit in the client and propose some changes.  I'll post what I find here for discussion.  Thanks.